r-package/grf/R/local_linear_forest.R

```diff
@@ -5,8 +5,8 @@
 #'
 #' @param X The covariates used in the regression.
 #' @param Y The outcome.
-#' @param ll.splits Optional choice to make forest splits based on ridge residuals as opposed to standard
-#'                  CART splits. Defaults to FALSE.
+#' @param enable.ll.split Optional choice to make forest splits based on ridge residuals as opposed to standard
```

Comment:
We've just been putting the experimental notice right after the variable name, for example 
```
#' @param sample.weights (experimental)
```